guides/hack/00-overview/03-async.md

```diff
@@ -1,85 +1,15 @@
 ### Async Programming in Hack

```

Comment:
So this is happening with the current user guide content then? Is the rendering wrong?